RileyLinkKit/ViewControllers/RileyLinkDeviceTableViewController.swift

```diff
@@ -286,13 +286,13 @@ public class RileyLinkDeviceTableViewController: UITableViewController {
 
                 self.showViewController(vc, sender: indexPath)
             case .MySentryPair:
-                let vc = self.storyboard!.instantiateViewControllerWithIdentifier(MySentryPairViewController.className) as! MySentryPairViewController
-                
-                vc.device = device
-                
-                vc.title = NSLocalizedString("MySentry Pair", comment: "Title of screen for pairing with MySentry.")
-                
-                self.showViewController(vc, sender: indexPath)
+                if let vc = self.storyboard?.instantiateViewControllerWithIdentifier(MySentryPairViewController.className) as? MySentryPairViewController {
```

Comment:
I'm going to remove the storyboard dependency from `RileyLinkKit` entirely, but for now I'd rather this fail silently for users without identical storyboard setups than crash.
If we felt like we really needed a storyboard/nib, then we should create a separate one and package it with the framework.